### DISCUSSION TITLE
How do you uninstall this on Windows?

### DISCUSSION THREAD

**Discussion by kazerniel** [ransome1/sleek#659] [Q&A] [answered]
I see that the app copied its files under User/.../AppData/Roaming, but not sure if it created files anywhere else. (It doesn't appear in the Apps & features menu or in the Start menu.)

Do I just delete that folder or is there anything else to remove?

**Comment by ransome1** [comment, marked as answer]
Yes, it should be enough to simply remove that folder.

The user data folders are documented here: https://github.com/ransome1/sleek/wiki/User-data-folder-location

**Reply by kazerniel** [reply]
Thank you!

**Reply by docnux** [reply]
During some deinstall-install-hazzle I discovered that there is a Sleek entry in the Windows registry. You may have to deal with it also.
